lms/djangoapps/learner_dashboard/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+Learner's Dashboard urls
+"""
+
+from django.conf.urls import patterns, url
+from django.conf import settings
+
+urlpatterns = patterns(
+    '',
+    url(r'^programs', 'learner_dashboard.views.view_programs', name="get_program_listing_view")
```

Comment:
Just name this "program_listing_view." You don't want to bake information about HTTP methods into the URL name.